shinysdr/devices.py

```diff
@@ -138,6 +138,12 @@ def close():
 __all__.append('IComponent')
 
 
+class ITopWatcher(Interface):
```

Comment:
This is not a good plan. `Top` is not a well-designed interface, and is going to go away in its current form when the multi-session refactoring finally happens.

Since I want this to happen anyway, I'll see if I can implement a better approach for hooking up telemetry.

Reply:
OK, commit 53f699e2d4705a7f4681bc09f1626b93509fd7ca adds a `DeviceContext` which can be used for this purpose, and commit 76afa0a589f35fcf73db3ac79514965934da1097 is an example of it being used for the existing `APRSISRXDevice`.